### ISSUE TITLE
When VideoPress block shows error it can still play video

### ISSUE THREAD

**Issue by lmischner** [wordpress-mobile/gutenberg-mobile#5758]
**Describe the bug**
If I get into a situation where the VideoPress block shows there's been an error, I can still tap the video to launch the WebView and play it. 

**To Reproduce**
1. (Not sure exactly how I got to this state -- it happened once when I turned off the phone screen while uploading a video)
2. Note error in VP block
3. Tap video
4. Note you can still play video from WebView

**Expected behavior**
If there's an error, I'd expect the video is not playable. If the video is playable, there shouldn't be an error

**Screenshots**
https://github.com/wordpress-mobile/gutenberg-mobile/assets/103850310/91113993-a7e0-48d9-b4d6-4fd815032967

**Smartphone (please complete the following information):**
 - Device: Pixel 7
 - OS: Android 13
 - Version VideoPress project installable build

**Comment by jhnstn** [comment]
I know I've seen this on iOS as well. I haven't traced back the issue but I'm fairly certain it's an error on VideoPress. 

It's probably an error with a single request to VideoPress. That would explain why the video does work when opening up the WebView. 

We could look into a "retry" flow if the video comes back with an error. I'm trying to remember if the iOS shows a retry message or not?

**Comment by fluiddot** [comment]
> We could look into a "retry" flow if the video comes back with an error.

Good idea. Although, if the error is in the player, we'd need to get notified by it somehow in order to display a `retry` button. I'm not sure if there's a VideoPress event for that 🤔 

> I'm trying to remember if the iOS shows a retry message or not?

I think we only show a retry message when the upload fails but not when the player fails.
